packages/desktop_drop/example/android/app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.desktop_drop_example"
-        minSdkVersion 16
+        minSdkVersion flutter.minSdkVersion
```

Comment:
This change has been automated by the Flutter CLI tool, though it's required to support Flutter 3.21. See [Dropping support for Android KitKat](https://docs.flutter.dev/release/breaking-changes/android-kitkat-deprecation).